### PR TITLE
Using docker within LXD

### PR DESCRIPTION
**Pull request by michal-at-travisci**
Added section on using docker within LXD incl. `sudo docker` usage examples

### CODE REVIEW

File: user/multi-cpu-architectures.md [hunk 1]
```diff
@@ -77,5 +78,36 @@ matrix:
    - os: linux
      env: LIB_PATH="/usr/bin/shared/x86_64/v2"
 ```
+{: data-file=".travis.yml"}
 
 For example, the above `.travis.yml`, would result in running both jobs with the environmental variable LIB_PATH assigned different values being run only on `amd64` architecture.
+
+## Using docker in `Arm` based builds within LXD container
```
Comment (Pezi777) [inline user/multi-cpu-architectures.md]:
```suggestion
## Using Docker in `Arm`-Based Builds within LXD Containers
```

File: user/multi-cpu-architectures.md [hunk 2]
```diff
@@ -77,5 +78,36 @@ matrix:
...
+
+It is possible to use Docker in `Arm` based builds within a LXD container. You may need an arm64v8 docker image as a base or ensure arm64 libraries required by your build are added to your Dockerfile. In order to use docker in `Arm` based build within LXD container, docker commands must be run with `sudo`.
```
Comment (Pezi777) [inline user/multi-cpu-architectures.md]:
```suggestion
It is possible to use Docker in `Arm`-based builds within an LXD container. You may need an arm64v8 docker image as a base or ensure arm64 libraries required by your build are added to your docker file. In order to use docker in `Arm`-based builds within LXD containers, docker commands must be run with `sudo`. 
```

File: user/multi-cpu-architectures.md [hunk 3]
```diff
@@ -77,5 +78,36 @@ matrix:
...
 ```
+{: data-file=".travis.yml"}
 
 For example, the above `.travis.yml`, would result in running both jobs with the environmental variable LIB_PATH assigned different values being run only on `amd64` architecture.
+
+## Using docker in `Arm` based builds within LXD container
+
+It is possible to use Docker in `Arm` based builds within a LXD container. You may need an arm64v8 docker image as a base or ensure arm64 libraries required by your build are added to your Dockerfile. In order to use docker in `Arm` based build within LXD container, docker commands must be run with `sudo`. 
+
+An example of building  docker image from Dockerfile adjusted to arm64:
```
Comment (Pezi777) [inline user/multi-cpu-architectures.md]:
```suggestion
An example of building a docker image from a docker file adjusted to arm64:
```

File: user/multi-cpu-architectures.md [hunk 4]
```diff
@@ -77,5 +78,36 @@ matrix:
...
+```yaml
+arch: arm64
+services:
+  - docker
+sudo: true
+script: sudo docker run my/test #assuming docker image my/test is arm64v8 ready
+```
+{: data-file=".travis.yml"}
+
+Please see also [Using Docker in Builds](user/docker/)
```
Comment (Pezi777) [inline user/multi-cpu-architectures.md]:
```suggestion
You can also have a look at [Using Docker in Builds](user/docker/).
```

File: user/multi-cpu-architectures.md (hunk 3 above)
Comment (michal-at-travisci) [reply user/multi-cpu-architectures.md]: @Pezi777 The `Dockerfile` is a own name, used throughout Docker documentation as well as in Travis - I would leave it as it is for coherency?

**Pull request merged**